### PR TITLE
Minor improvements to asset_id_files, let it kinda run for PAL

### PR DESCRIPTION
**Pull request by henriquegemignani**
But not really because FrontEnd has a different asset id. 😢

### CODE REVIEW

**Review by duncathan — COMMENTED** [review]
honestly, FrontEnd may warrant special handling altogether. it only has the one area in it - maybe we should just write the front-end file by hand?

**Comment by henriquegemignani** [comment]
Changed to skip entirely creating files for FrontEnd and the Multiplayer paks.

Whenever we change FrontEnd, we can handle that directly since it has entirely different ids (and will need to check if the areas are very different or not?!)

File: src/open_prime_rando/echoes/asset_ids/__init__.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
-from . import agon_wastes, great_temple, sanctuary_fortress, temple_grounds, torvus_bog, world
+from . import agon_wastes, great_temple, sanctuary_fortress, temple_grounds, torvus_bog
 
 __all__ = [
-    "agon_wastes", "great_temple", "sanctuary_fortress", "temple_grounds", "torvus_bog", "world",
```
Comment (duncathan) [inline src/open_prime_rando/echoes/asset_ids/__init__.py:4]: why remove world?

Comment (henriquegemignani) [reply src/open_prime_rando/echoes/asset_ids/__init__.py:4]: I made this file generated and I did an oopsie

**Review by duncathan — APPROVED** [review]

**Pull request merged**